File: programmers/2021_02/day04.js
function solution(n) {
    var answer = '';
    
    // 반복문: n이 0이 되면 탈출한다.
    while (n > 0) {
        // 조건문: n을 2로 나눠서 0이 아니면 수, 0이면 박을 입력한다.
        if (n % 2 != 0) {
            answer = '수' + answer;
        } else {
            answer = '박' + answer;
        }
        // 조건문이 끝나면, n에서 -1를 한다.
        n--;
    }
    
    return answer;
}

File: programmers/2021/month10/day04.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;

class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        ArrayList<Integer> list1 = new ArrayList<>();
        ArrayList<Integer> list2 = new ArrayList<>();
        for (int i = 0; i < sizes.length; i++) {
        	int getNum1 = 0;
        	int getNum2 = 0;
        	getNum1 = sizes[i][0];
        	for (int j = 0; j < sizes[i].length; j++) {
        		getNum2 = sizes[i][j];
        	}
        	if (getNum1 > getNum2) {
        		list1.add(getNum1);
        		list2.add(getNum2);
        	} else {
        		list1.add(getNum2);
        		list2.add(getNum1);
        	}
        }
        list1.sort(Comparator.naturalOrder());
        Collections.sort(list2);
        int getLastNum1 = list1.get(list1.size() - 1);
        int getLastNum2 = list2.get(list2.size() - 1);
        answer = getLastNum1 * getLastNum2;
        return answer;
    }
}
